rosetta-lang/src/main/java/com/regnosys/rosetta/types/RObjectFactory.java

```diff
@@ -217,4 +213,10 @@ public RDataType buildRDataType(Data data, List<RAttribute> additionalAttributes
 	public REnumType buildREnumType(RosettaEnumeration enumeration) {
 		return new REnumType(enumeration, modelIdProvider, this);
 	}
+	public RMetaAnnotatedType withMeta(RType rtype, List<RMetaAttribute> metaAttributes) {
+		return new RMetaAnnotatedType(rtype, metaAttributes);
+	}
+	public RMetaAnnotatedType withEmptyMeta(RType rtype) {
+		return new RMetaAnnotatedType(rtype, List.of());
+	}
```

Comment:
I wonder if these two would be better placed as factory methods on `RMetaAnnotatedType`.

Reply:
done